tests/application/test_operator.py

```diff
@@ -95,12 +93,12 @@ def test_bond_operator(accounts, threshold_staking, pre_application, chain):
     assert all_locked == 0
     assert len(staking_providers) == 0
 
-    pre_application.confirmOperatorAddress({"from": operator1})
+    pre_application.confirmOperatorAddress(sender=operator1)
     assert pre_application.stakingProviderInfo(staking_provider_3)[CONFIRMATION_SLOT]
     assert pre_application.isOperatorConfirmed(operator1)
 
-    assert "OperatorBonded" in tx.events
-    events = tx.events["OperatorBonded"]
+    events = pre_application.OperatorBonded.from_receipt(tx)
+    assert len(events) == 1
```

Comment:
```suggestion
    assert len(tx.events.filter(pre_application.OperatorBonded)) == 1
```

But look out for 0.6.8 for a better events assertion workflow like this:

```py
assert tx.events == [pre_application.OperatorBonded()]
```

Reply:
`pre_application.OperatorBonded` returns event type. How to make an instance of event? I tried `pre_application.OperatorBonded()` but getting an error that `TypeError: 'ContractEvent' object is not callable`

Reply:
The filter approach is the best we can do until the functionality around initializing events is added (hopefully in 0.6.9, sorry I misspoke when I said 0.6.8)